routing.rst

```diff
@@ -1262,9 +1267,6 @@ the common configuration using options when importing the routes.
 
                 // these requirements are added to all imported routes
                 ->requirements(['_locale' => 'en|es|fr'])
-
-                // you can optionally exclude some files/subdirectories when loading annotations
```

Comment:
Could you please keep this command by moving it above the extra arg?

Reply:
You want that on the line below 

```
            // use the optional fourth argument of import() to exclude some files
            // or subdirectories when loading annotations
```

Because having 2 sentences describe the exclusion argument sounds a bit redundant.

Reply:
I agree that it would be repetitive, so I merged this as is. Thanks.